chatwoot/chatwoot
# fix: Issue on the status filter field

## Description

If I add more tags in the multi-select field, the tags are overlapped and cause UI issue. This PR fix the UI issue by removing some classes and height from multi-select.

Fixes https://github.com/chatwoot/chatwoot/issues/7440#event-10469856785

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

- Run the app and go to conversations
- Click on filter conversations. It will open a modal
- Select more tags then you see tags are overlapped.

## Checklist:


## Demo

https://www.loom.com/share/0395ffb3a89748058e844f0090b89105
